src/js/components/stock-movement-wizard/outbound/EditPage.jsx

```diff
@@ -441,6 +449,18 @@ class EditItemsPage extends Component {
     return Promise.resolve();
   }
 
+  updateRow(values, index) {
+    const item = values.editPageItems[index];
```

Comment:
Does this problem occur only on edit page items on outbound sm? How about add items pages after save or when you have stock list?

Reply:
@awalkowiak You're right. I've added this update method to add items page in all workflows.